test/src/commands/create/create_test.dart

```diff
@@ -535,6 +541,7 @@ void main() {
               vars: <String, dynamic>{
                 'project_name': 'my_app',
                 'org_name': 'com.example.verygoodcore',
+                'executable_name': 'my_app',
```

Comment:
Is not required for this tests right?

Reply:
It is, otherwise, the verification doesn't match.